gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -695,6 +695,10 @@ public static BlobListOption prefix(String prefix) {
 
     /**
      * Returns an option to specify whether blob listing should include subdirectories or not.
+     * {@link StorageOptions#pathDelimiter()} is used as a path delimiter. If set to {@code true}
+     * also blobs in subdirectories are listed. If set to {@code false} and used in combination
```

Comment:
I am not sure I understand the "and used in combination" part. delimiter (recursive false) can be used without prefix and should return all the blobs in the "root" folder (blobs name without any delimiter).